deeplake/core/index_maintenance.py

```diff
@@ -29,16 +29,10 @@ def is_embedding_tensor(tensor):
     )
 
 
-def validate_embedding_tensor(tensor):
+def validate_text_tensor(tensor):
     """Check if a tensor is an embedding tensor."""
-
-    valid_names = ["embedding"]
-
-    return (
-        tensor.htype == "embedding"
-        or tensor.meta.name in valid_names
-        or tensor.key in valid_names
-    )
+    p = tensor.htype
```

Comment:
Let's remove this extra assignment just check `tensor-htype == "text"`

Reply:
Done.